src/VisualStudio/Core/Def/Implementation/CodeCleanup/CodeCleanupFixerProvider.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.ComponentModel.Composition;
+using Microsoft.VisualStudio.Language.CodeCleanUp;
+using Microsoft.VisualStudio.Utilities;
+
+namespace Microsoft.VisualStudio.LanguageServices.Implementation.CodeCleanup
+{
+    [Export(typeof(ICodeCleanUpFixerProvider))]
```

Comment:
is ICodeCleanupFixerProvider new editor interface? is it okay to export lie this? won't it to cause us to be loaded when C#/VB project is not actually loaded in VS?

Reply:
Resolved and documented now